<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticDialog</class>
 <widget class="QDialog" name="StatisticDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Your statistics</string>
  </property>
  <widget class="QGroupBox" name="generalStatisticGroup">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>General statistic</string>
   </property>
   <widget class="QLabel" name="allRecordsLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <property name="text">
     <string>ALL RECORDS</string>
    </property>
   </widget>
   <widget class="QLabel" name="allRecordsValue">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>28</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="productiveLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PRODUCTIVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="wastedLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>72</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WASTED</string>
    </property>
   </widget>
   <widget class="QLabel" name="productiveValue">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>50</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="wastedValue">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="efficienceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>EFFICIENCY</string>
    </property>
   </widget>
   <widget class="QLabel" name="efficienceValue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>200</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/icons/man-success.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>50</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/icons/bubble.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="statusText">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start activity :)</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
